dev/com.ibm.websphere.appserver.features/visibility/public/dataContainer-1.0/resources/l10n/io.openliberty.dataContainer-1.0.properties

```diff
@@ -0,0 +1,24 @@
+###############################################################################
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=Jakarta Data is a programming model for data access that\
+ allows you to represent data with simple Java objects called entities\
+ and define operations on data as methods of a repository interface.\
+ A Jakarta Data provider supplies the implementation.\
+ This feature only includes the Jakarta Data 1.0 specification interfaces.\
+ This feature does not include a Jakarta Data provider.
```

Comment:
Jakarta Data is a programming model for data access. 
You can use Jakarta Data to represent data with Java objects called entities and to define operations in data as methods of a repository interface.
A Jakarta Data provider supplies the repository implementation to an application.
The dataContainer-1.0. feature only includes the Jakarta Data 1.0 specification interfaces.
The dataContainer-1.0. feature does not include a Jakarta Data provider.